pynucastro/constants/test/test_constants.py

```diff
@@ -0,0 +1,31 @@
+import math
+from pytest import approx
+
+from pynucastro.constants import constants
+
+
+class TestConstants:
+    def test_constants(self):
+
+        assert constants.m_u == approx(1.66054e-24)
+        assert constants.m_u_MeV == approx(constants.m_u * constants.c_light**2 * constants.erg2MeV)
+        assert constants.m_p == approx(1.67262192e-24)
+        assert constants.m_p_MeV == approx(constants.m_p * constants.c_light**2 * constants.erg2MeV)
+        assert constants.m_n == approx(1.674927498e-24)
+        assert constants.m_n_MeV == approx(constants.m_n * constants.c_light**2 * constants.erg2MeV)
+        assert constants.m_e == approx(9.1093837e-28)
+        assert constants.m_e_MeV == approx(constants.m_e * constants.c_light**2 * constants.erg2MeV)
+
+        assert constants.N_A == approx(6.02214e23)
+
+        assert constants.k == approx(1.30806e-16)
+        assert constants.k_MeV == approx(constants.k * constants.erg2MeV)
+
+        assert constants.h == approx(6.62607e-27)
+        assert constants.h == approx(6.62607e-27 / 2 / math.pi)
```

Comment:
This should have made the test fail. I think we need to pass `abs=0.0` to all the `approx` calls, since we're never comparing to 0.
```suggestion
        assert constants.hbar == approx(6.62607e-27 / 2 / math.pi)
```